data/txts/tips/amazons.lua

```diff
@@ -1,15 +1,15 @@
 push_textdomain("texts")
 tips = {
    {
-      text = _"Amazons love trees. Build enough jungle preserver's huts to have enough of them.",
+      text = _"Amazons love trees. Build enough jungle preserver’s huts to have enough of them.",
       seconds = 6
    },
    {
-      text = _"Rare woods are the backbone of an amazon economy. Build rare tree plantations to plant these special trees.",
+      text = _"Rare trees are the backbone of an amazon economy. Build rare tree plantations to plant these special trees.",
```

Comment:
I'd rather prefer to keep the terminology woods here, anmd change the second sentence to read: ".. to plant the special trees for these woods." or ".. to plant the special trees to produce them."